src/main/java/org/example/ByteValidator.java

```diff
@@ -0,0 +1,13 @@
+package org.example;
+
+public class ByteValidator {
+    public static boolean isValidByteArray(byte[] byteArray, String validCharacters) {
```

Comment:
The second argument must not be a string of valid characters. It must be the name of **Character set**

I want to test if the byte array is a valid string in that character set. Check out `CharacterSet` class in java: